src/document/json/rht_pq_map.ts

```diff
@@ -74,7 +75,29 @@ export class RHTPQMap {
       return null;
     }
 
-    this.nodeMapByCreatedAt.get(createdAt.toIDString()).remove(executedAt);
+    const node = this.nodeMapByCreatedAt.get(createdAt.toIDString());
+    node.remove(executedAt);
+    return node.getValue();
```

Comment:
It seems there was a problem with the previous implementation. Thanks for the correction. 👍